Vic2ToHoI4/Source/HOI4World/Map/MapData.h

```diff
@@ -0,0 +1,55 @@
+#ifndef MAP_DATA_H
+#define MAP_DATA_H
+
+
+
+#include "ProvincePoints.h"
+#include <optional>
+#include <map>
+#include <set>
+#include "../../../../bitmap/bitmap_image.hpp"
```

Comment:
You should probably shove this into VS project's include, seeing how it's outside the project.